WordPress/Tracks+ShareExtension.swift

```diff
@@ -43,6 +43,15 @@ extension Tracks {
         trackExtensionEvent(.categoriesSelected, properties: properties as [String: AnyObject]?)
     }
 
+    public func trackExtensionPostTypeOpened() {
+        trackExtensionEvent(.postTypeOpened)
+    }
+
+    public func trackExtensionPostTypeSelected(_ postType: String) {
+        let properties = ["post_type": postType]
+        trackExtensionEvent(.postTypeSelected, properties: properties as [String: AnyObject]?)
```

Comment:
I think the properties dictionary doesn't need to be cast as an optional.  What's the thinking behind that choice?

Reply:
No reason, it's a copy/pasted from the other track methods in the file. You're right, it's unnecessary. I'll clean up the other ones in there, too.